projects/demo/src/modules/components/progress-bar/examples/1/index.html

```diff
@@ -0,0 +1,6 @@
+<progress
+    tuiProgressBar
+    [value]="value$ | async"
```

Comment:
We put bindings after plain string attributes in our code style

Reply:
fixed